src/Fixer/DoctrineMigrationFixer.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace AppBundle\Fixer;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class DoctrineMigrationFixer implements DefinedFixerInterface
+{
+    public function getName(): string
+    {
+        return 'PhpCsFixerCustomFixer/doctrine_migration_clean';
+    }
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Migrations files should not have declare(strict_types=1), auto-generated comments, and abortif',
```

Comment:
Not a big deal, but what do you think about : 
`..., auto-generated comments, and abortIf() calls',`